dotnet/html-formatter/htmlformatter/htmlformatter.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <LangVersion>10.0</LangVersion>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Remove="Resources\index.mustache.html" />
+    <None Remove="Resources\main.css" />
+    <None Remove="Resources\main.js" />
+    <None Remove="Resources\main.js.LICENSE.txt" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Include="Resources\index.mustache.html" />
+    <EmbeddedResource Include="Resources\main.css" />
+    <EmbeddedResource Include="Resources\main.js" />
+    <EmbeddedResource Include="Resources\main.js.LICENSE.txt" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Cucumber.Messages" Version="27.0.2" />
```

Comment:
Is it possible to use a version range here? The html formatter is fairly backwards compatible with older messages.

Reply:
I put in a range to go back to R26. That was when I updated the Messages code generation approach to use the current system (retiring the old protobuf method). Assuming there was a risk that I broke something there, that is as far back as I'm comfortable.

Reply:
I had to correct that to only r27.0 as the minimum. R27 introduced a new enumeration type (HookType) and the serializer needs to know about all such enumerations for proper conversion.